### DISCUSSION TITLE
GPAccruals

### DISCUSSION THREAD

**Discussion by RoboMWM** [GriefPrevention/GriefPrevention#2075] [Addons]
### GPAccruals
Control claim block accrual rates and limits for groups of players, based on permissions.

[Download](https://www.dropbox.com/s/5ilyqmkydbp0yhh/GPAccruals-1.1.jar?dl=0) | [Documentation + support](https://bitbucket.org/BillyGalbreath/gpaccruals/src/master/src/main/resources/config.yml)

**Comment by RoboMWM** [comment]
Note: I do not maintain this plugin, see links above for support.

**Comment by QarthO** [comment]
Note: Even though this plugin hasn't been updated in years, its <ins>still fully functioning on the latest version of GP</ins>. 
> As of writing this, GPAccruals works from Minecraft Versions 1.11.2 -> 1.21.1. 
> And is expected to continue working on future versions.

The original author GPAccruals no longer maintains the plugin and has given me permission to take over it. 
![image](https://github.com/user-attachments/assets/caf1f2bc-f7d3-458d-96e0-27bbc6fa9331)

For any support, bugs, or if Bukkit/GP API updates cause the plugin to break, message me in my [discord](https://quartzdev.gg/discord)
